### PR TITLE
1 of 2 DBs seeded

### PR DESCRIPTION
**Pull request by dev-poston**
@HannahManfredi or @jametevia, could you please review this seeding script, thank you!

### CODE REVIEW

**Comment by dev-poston** [comment]
ignore the console log :|

File: couchDBSeeder.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+let seedScript = (dbCount = 1) => {
+  let dbCounter = dbCount;
+  let bulk = {
+    docs: []
+  };
+  for (var i = 0; i < 5000; i++) {
+    bulk.docs.push({productId: dbCounter, images: ['https://picsum.photos/200', 'https://picsum.photos/200', 'https://picsum.photos/200']});
+    dbCounter++;
+  }
+  axios.post(url, bulk)
```
Comment (hannahmanfredi) [inline couchDBSeeder.js:15]: woah this is tight! so you're seeding batches of 1k records via post req to your couch db?!

**Review by hannahmanfredi — APPROVED** [review]
Quick review. Grea job, all looks good. I also use a config file for secret passwords, etc. CouchDb seems really cool!

**Comment by hannahmanfredi** [comment]
Last thing I forgot to add is that I'd love to see a test or two for your couch seeding script.

File: couchDBSeeder.js (hunk 1 above)
Comment (dev-poston) [reply couchDBSeeder.js:15]: @HannahManfredi batches of 5k (ignore the console log), script took less than 10min.

**Pull request merged**